addons/website/static/src/scss/website.scss

```diff
@@ -1246,6 +1257,59 @@ header {
     }
 }
 
+// Offcanvas menu on mobile or offcanvas themed header override
+.o_mobile_or_offcanvas {
+    .o_mega_nav, .o_mega_menu {
+        max-width: $offcanvas-horizontal-width;
+        transition: transform .3s ease-in-out, visibility .3s ease-in-out;
+        transform: translateX(var(--FakeOffcanvas-animation, 100%));
+    }
+
+    .o_mega_menu.show, .o_mega_menu.show ~ .o_mega_nav {
+        visibility: visible !important;
+        transform: translateX(0%);
+        transition: transform .3s ease-in-out, visibility .3s ease-in-out;
```

Comment:
Better to use BS [default mixin](https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/mixins/_transition.scss#L2).  

This way we'll ensure:
- Consistency with other BS components in case the user overrides `$enable-transitions` to false
- To be compliant with user preferences (see `$enable-reduced-motion`);